bin/createRelease.js

```diff
@@ -0,0 +1,71 @@
+#! /usr/bin/env node
+const chalk = require('chalk')
+const { Octokit } = require('@octokit/rest')
+
+const { readConfig } = require('../lib/config')
+const { GitHubService } = require('../lib/github')
+const { composeReleaseBody, defaultFormatter, defaultCategorizer } = require('../lib/release')
+
+const NOT_FOUND_MESSAGE = `
+GitHub API returned 404 Not Found when creating the new release. This probably means two things:
+  1. You have valid API Token
+  2. However, he user doesn't have sufficient permissions on the repository. So please make sure that they are at least a "Maintainer".
+`.trim()
+
+async function main () {
+  const config = getConfig()
+
+  try {
+    const gh = new GitHubService(
+      new Octokit({ auth: config.gitHubToken }),
+      config.repoOwner,
+      config.repoName)
```

Comment:
`NIT` I think we could add some lint to this project in the future.

Reply:
It does use standard. But this line is awkward indeed.